var/spack/repos/builtin/packages/pexsi/package.py

```diff
@@ -85,20 +95,29 @@ def install(self, spec, prefix):
         for key, value in substitutions.items():
             filter_file(key, value, makefile)
 
-        make()
+    def build(self, spec, prefix):
+        super(Pexsi, self).build(spec, prefix)
```

Comment:
I would just run `make()` here. The super class doesn't do anything fancy. I think it would be more obvious how things work if you did that.